test/deployframework/context.go

```diff
@@ -120,8 +121,31 @@ func (df *DeployFramework) NewDeployerCtx(
 		targetPodCount -= 2
 	}
 
-	df.Logger.Debugf("Deployer config: %+v", cfg)
-	deployer, err := deploy.NewDeployer(*cfg, df.Logger, df.Client, df.APIExtClient, df.MeteringClient, df.OLMV1Client, df.OLMV1Alpha1Client)
+	logger := logrus.New()
+	// with Json Formatter
+	logger.Formatter = &logrus.JSONFormatter{}
+	logger.SetOutput(os.Stdout)
+
+	outFile, err := os.OpenFile(filepath.Join(outputPath, "test-output.log"), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0755)
+	if err != nil {
+		logrus.Fatalf("failed to create the logger for this ctx: %v", err)
+	}
+	// defer outFile.Close()
+
+	logger.SetOutput(outFile)
+	logger.SetLevel(logrus.DebugLevel)
+	logger.SetFormatter(&logrus.TextFormatter{
+		FullTimestamp:   true,
+		TimestampFormat: "01-02-2006 15:04:05",
+		ForceColors:     true,
+	})
+
+	outLogger := logger.WithFields(logrus.Fields{
+		"context": namespace,
+	})
```

Comment:
I need to clean this up a bit, but the general idea is that for an individual Metering installation, instead of logging everything to stdout using the same logger as the `deployframework` structure, we create another instance logger and re-direct it's output to a file.